app/views/trips/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p class="title"><%= t(".title") %></p>
+<table>
+  <%= render "trips_table_header" %>
+  <%= render "trip" %>
+  <td class="title">Still to allocate</td>
```

Comment:
translation reminder when you get to them

Reply:
But yes, I need translation here, you are right, I missed it (probably more), thanks 😄